package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "prepare": "husky install",
     "compile-assets": "grunt",
-    "watch-node": "ts-node-dev --respawn server.ts",
+    "watch-node": "ts-node-dev --watch .env --respawn server.ts",
```

Comment:
Not related to the change, but handy for the server to automatically restart when config is updated.